### PR TITLE
feat: add loading animations to the vote page

### PR DESCRIPTION
**Pull request by hetfly**

### CODE REVIEW

**Review by parrot9design — APPROVED** [review]
This is fantastic, thanks @hetfly!

**Comment by WeiLi512** [comment]
great work!

File: src/app/components/ProposalInfo/index.tsx [hunk 1]
```diff
@@ -54,54 +55,81 @@ const ContentText = styled(Typography)`
...
-    for: approvePercentage,
-    against: rejectPercentage,
-    uniqueApproveVoters,
-    uniqueRejectVoters,
-  } = proposal;
+export const StyledSkeleton = styled(Skeleton)`
+  background-color: rgba(44, 169, 183, 0.2) !important;
+`;
+
+export default function ProposalInfo({ proposal }: { proposal: ProposalInterface | null }) {
```
Comment (WeiLi512) [inline src/app/components/ProposalInfo/index.tsx:62]: why did you refactor?

**Review by WeiLi512 — CHANGES_REQUESTED** [review]

File: src/app/components/ProposalInfo/index.tsx [hunk 2]
```diff
@@ -54,54 +55,81 @@ const ContentText = styled(Typography)`
...
+  const title = proposal?.name;
+  const description = proposal?.description;
+  const startDay = proposal?.startDay;
+  const endDay = proposal?.endDay;
```
Comment (WeiLi512) [inline src/app/components/ProposalInfo/index.tsx:66]: why did you refactor the code?

**Review by WeiLi512 — CHANGES_REQUESTED** [review]

File: src/app/components/ProposalInfo/index.tsx (hunk 2 above)
Comment (hetfly) [reply src/app/components/ProposalInfo/index.tsx:66]: I couldn't properly destructure the props after allowing them to be null

**Comment by hetfly** [comment]
@WeiLi512 this looks better, thanks

**Pull request merged**